js/lex.go

```diff
@@ -289,35 +288,35 @@ func (l *Lexer) consumeUnicodeEscape() bool {
 	return true
 }
 
-////////////////////////////////////////////////////////////////
-
-func (l *Lexer) consumeSingleLineToken(skip int) {
-	l.r.Move(skip)
+func (l *Lexer) consumeSingleLineComment() {
 	for {
 		c := l.r.Peek(0)
 		if c == '\r' || c == '\n' || c == 0 {
 			break
 		} else if c >= 0xC0 {
-			mark := l.r.Pos()
+			//mark := l.r.Pos()
 			if r, _ := l.r.PeekRune(0); r == '\u2028' || r == '\u2029' {
-				l.r.Rewind(mark)
+				//l.r.Rewind(mark)
 				break
 			}
 		}
 		l.r.Move(1)
```

Comment:
Sure, it's more like optimization - if we anyway peeked the rune up, we might as well use the already-known length to skip it entirely instead of continuing the loop inside of it.